inference-engine/src/legacy_api/src/transformations/convert_opset1_to_legacy/convert_one_hot_to_one_hot_ie.cpp

```diff
@@ -46,7 +46,8 @@ ngraph::pass::ConvertOneHotToOneHotIEMatcher::ConvertOneHotToOneHotIEMatcher() {
         // insert Convert layer to cast output to a correct data type defined by the on/off values
         if (on_value_node->get_element_type() != m_output_type) {
             auto convert = std::make_shared<ngraph::opset1::Convert>(one_hot_ie, on_value_node->get_element_type());
-            convert->set_friendly_name(one_hot->get_friendly_name() + "/Convert");
+            convert->set_friendly_name(one_hot->get_friendly_name());
+            one_hot->set_friendly_name(one_hot->get_friendly_name() + "/FloatOutput");
```

Comment:
We usually do not set the names to intermediate nodes as it could lead to non unique names.